ddtrace/_monkey.py

```diff
@@ -84,24 +82,22 @@
     "dogpile_cache": True,
     "yaaredis": True,
     "asyncpg": True,
+    "tornado": False,
 }
 
+
+CONTRIB_DEPENDENCIES = {
```

Comment:
It's weird that this information can't be included in the `tornado` contrib module itself, because that would mean accessing it would require an import. It makes sense the way you have it. An explanatory comment or more detailed variable name could clarify why this is necessary.